content/dstu2/person.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Person | DSTU 2 API
+---
+
+# Person
+
+* TOC
+{:toc}
+
+## Terminology Bindings
+
+<%= terminology_table(:person, :dstu2) %>
+
+## Search
+
+Search for Persons that meet supplied query parameters:
+
+    GET /Person?:parameters
+
+### Parameters
+
+ Name         | Required?                                                    | Type       | Description
+--------------|--------------------------------------------------------------|------------|------------------------------------------------------------------------------------
+ `_id`        | No, if populated all other parameters are ignored            | [`token`]  | The logical resource id associated with the resource.
+ `identifier` | Yes, or `_id`                                                | [`token`]  | The person identifier.  Example: `urn:oid:2.16.840.1.113883.3.13.6|01022228`
```

Comment:
Could trim out some of the empty space in this table to keep the style consistent with other resources :)

```
 Name         | Required?                                         | Type      | Description
--------------|---------------------------------------------------|-----------|-------------------------------------------------------
 `_id`        | No, if populated all other parameters are ignored | [`token`] | The logical resource id associated with the resource.
 `identifier` | Yes, or `_id`                                     | [`token`] | The person identifier.  Example: `01022228`
```